src/main/java/org/vandeseer/easytable/structure/cell/AbstractCell.java

```diff
@@ -87,6 +100,23 @@ public float getBorderWidthBottom() {
                 : 0;
     }
 
+
+    public boolean hasBorderStyleTop() {
+        return getBorderStyleTop() !=null && getBorderStyleTop() == BorderStyle.DOTTED;
```

Comment:
I think it can't ever be `null` because it's explicitly set to `BorderStyle.SOLID` above, isn't it? 🤔

Reply:
Yep, It can never be null. you are right.